roles/openshift_health_checker/test/kibana_test.py

```diff
@@ -68,7 +65,7 @@ def assert_error(error, expect_error):
     ),
 ])
 def test_check_kibana(pods, expect_error):
-    check = canned_kibana()
+    check = kibana()
```

Comment:
Similar to another review today, maybe this is just `Kibana()` and no need for a fixture.